src/targets/php/helpers.ts

```diff
@@ -1,16 +1,21 @@
+import { escapeString } from '../../helpers/escape';
+
 export const convertType = (obj: any[] | any, indent?: string, lastIndent?: string) => {
   lastIndent = lastIndent || '';
   indent = indent || '';
 
   switch (Object.prototype.toString.call(obj)) {
+    case '[object Boolean]':
+      return obj;
```

Comment:
Any JSON object with a boolean in here was having that boolean rewritten to `null`. woof